packages/pixeloven/cli/src/commands/init.test.ts

```diff
@@ -0,0 +1,51 @@
+import "jest";
+
+import { build, print, prompt } from "gluegun";
+import { resolve } from "path";
+import sinon from "sinon";
+
+const cli = build()
+    .brand("pixeloven")
+    .src(resolve(__dirname, ".."))
+    .create();
+
+const Sandbox = sinon.createSandbox();
+
+const Stub = {
+    print: Sandbox.stub(print),
+    process: {
+        exit: Sandbox.stub(process, "exit"),
+    },
+    prompt: Sandbox.stub(prompt),
+};
+
+/**
+ * @todo https://shift.infinite.red/integration-testing-interactive-clis-93af3cc0d56f
+ * @todo Can't test this since templates is not able to be mocked.
+ *         - https://github.com/ductiletoaster/gluegun/blob/master/src/index.ts#L33
+ *         - Alternative here would be to mock out the file system and make this closer to a true integration test
+ */
+describe("@pixeloven/cli", () => {
+    describe("commands", () => {
```

Comment:
Do we need all the nesting here vs including commands and init in the opening describe wrapper?

Reply:
I did this everywhere so we can clean it up later but yeah it could be simplified.